.github/workflows/CI.yml

```diff
@@ -2,9 +2,7 @@ name: Current Integration
 
 on:
   push:
-    branches: [ gha ]
   pull_request:
-    branches: [ gha ]
```

Comment:
Doesn't this change mean that it'll never run on any pushes or PRs?

Reply:
That was the old version.
If you check the "Update CI.yml", you'll see that I removed the branches, so it will run on ANY push or pull request

Reply:
Yeah no, Master only.

On any PR is fine, but branch wise I don't want a build if i'm working on the dev branch for example. This means when I want to push changes just to PR a merge into master, we're launching 3 identical builds.

Once on the push to dev.
Once on the PR created.
Once on the merge commit to master.